src/client/src/App/Analytics/Positions/__tests__/Positions.test.tsx

```diff
@@ -119,33 +119,4 @@ describe("Positions", () => {
 
     expect(screen.getByTestId("tooltip").textContent).toBe("AUD -1,557,031")
   })
-
-  // TODO (5350): fix test failure - Windows snapshot incompatible with Mac
-  it.skip("should display the correct bubble chart", async () => {
```

Comment:
Since the aim was to generate a snapshot, I have removed it as @algreasley suggested that the test should be focused on component testing for user events such as clicking at a button, etc..